### PR TITLE
Rework homepage headlines

### PR DESCRIPTION
**Pull request by davidfischer**

### CODE REVIEW

File: ethicalads-theme/templates/ea/homepage.html [hunk 1]
```diff
@@ -24,12 +24,13 @@
 
         <!-- Heading -->
         <h1 class="display-3 font-weight-light" data-aos="fade-left">
-          <span class="d-block">Respectful Advertising</span>
-          <span class="d-block" data-aos-delay="100">For Developers</span>
+          <span class="d-block">Transparent Advertising</span>
+          <span class="d-block">For Developers</span>
+          <span class="d-block" data-aos-delay="100">That Works</span>
```
Comment (davidfischer) [inline ethicalads-theme/templates/ea/homepage.html:29]: While I don't hate this H1, I'm not sure it paints us enough as the guide. It's hard to get an H1 that works for both publishers and advertisers though.

Comment (ericholscher) [reply ethicalads-theme/templates/ea/homepage.html:29]: Yea, agreed. This seems better at least.

File: ethicalads-theme/templates/ea/homepage.html [hunk 2]
```diff
@@ -24,12 +24,13 @@
...
         </h1>
 
         <!-- Text -->
-        <p class="font-size-lg" data-aos="fade-left" data-aos-delay="200">EthicalAds is a privacy-preserving ad network targeting developers</p>
+        <p class="font-size-lg" data-aos="fade-left" data-aos-delay="200">For everyone tired of ads that track you, EthicalAds serves privacy-preserving ads to developers</p>
```
Comment (ericholscher) [inline ethicalads-theme/templates/ea/homepage.html:33]: This feels pretty publisher-focused and possibly offputting to advertisers. 

```suggestion
        <p class="font-size-lg" data-aos="fade-left" data-aos-delay="200">We get hand-picked advertisers in front of developers with privacy-preserving ads on our high-quality network</p>
```

Perhaps a bit overly verbose though.

**Review by ericholscher — APPROVED** [review]
H1 changes look good.

**Pull request merged**